src/frontend/app/shared/shared.module.ts

```diff
@@ -163,7 +163,7 @@ import { CfStacksCardComponent } from './components/list/list-types/cf-stacks/cf
     CardCfSpaceUsageComponent,
```

Comment:
Unrelated to this PR, but if we have a cloud foundry module these cf specific components should be listed there